src/lang.js

```diff
@@ -63,7 +63,7 @@
             return key;
         }
 
-        var message = this._getMessage(key);
+        var message = this._getMessage(key, replacements);
```

Comment:
@nivla you are right, I think I made a mistake because... git is blaming myself on that line: https://github.com/nivla/Lang.js/blame/revert-14-revert-13-master/src/lang.js#L66 :disappointed: